options/locale/locale_en-US.ini

```
@@ -2794,6 +2794,7 @@ repos.size = Size
 packages.package_manage_panel = Package Management
 packages.total_size = Total Size: %s
 packages.unreferenced_size = Unreferenced Size: %s
+packages.cleanup = Cleanup expired data
```

```suggestion
packages.cleanup = Clean up expired data
```

Verb should use a space as per https://writingexplained.org/cleanup-or-clean-up-difference